sdk/package.json

```diff
@@ -1,26 +1,26 @@
 {
   "name": "@stackblitz/sdk",
   "version": "1.6.0",
-  "description": "",
+  "description": "SDK for generating and embedding StackBlitz projects.",
```

Comment:
@sulco Does this description work?

Reply:
It might imply that embedding is the only thing you can do to the generated project, but tbh I'm not sure how we could rephrase it without sounding clunky